vowpalwabbit/binary.h

```diff
@@ -9,6 +9,6 @@ namespace VW
 {
 namespace binary
 {
-VW::LEARNER::base_learner* binary_setup(VW::config::options_i& options, vw& all);
+VW::LEARNER::base_learner* binary_setup(setup_base_fn& setup_base_fn, VW::config::options_i& options, vw& all);
```

Comment:
```suggestion
VW::LEARNER::base_learner* binary_setup(setup_base_fn& setup_base, VW::config::options_i& options, vw& all);
```
nit: arg name consistency.